app/server/lib/ICreate.ts

```diff
@@ -30,8 +30,9 @@ export interface ICreate {
 
   deploymentType(): GristDeploymentType;
   sessionSecret(): string;
-  // Check configuration of the app early enough to show on startup.
   configure?(): Promise<void>;
+  // Check configuration of the app early enough to show on startup.
```

Comment:
It would be good for the docstrings on configure and checkBackend to be distinct. Perhaps `checkBackend` comment could say `optionally perform sanity checks on the configured storage, throwing a fatal error if it is not functional` or something like that.

Reply:
Your comment seems good to me. And I placed this comment again above `configure?()`.